game-core/src/test/java/games/strategy/ui/ImageScrollerLargeViewTest.java

```diff
@@ -0,0 +1,34 @@
+package games.strategy.ui;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.closeTo;
+import static org.hamcrest.Matchers.is;
+
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+
+final class ImageScrollerLargeViewTest {
+  @Nested
+  final class NormalizeScaleTest {
+    static final double ERROR = 1.0E-9;
+
+    @Test
+    void shouldDiscretizeEachUnitInto256Values() {
+      assertThat(ImageScrollerLargeView.normalizeScale(1.0), is(closeTo(1.0, ERROR)));
+      assertThat(ImageScrollerLargeView.normalizeScale(0.71), is(closeTo(0.70703125, ERROR)));
```

Comment:
Okay, realizing now we are looking for multiples of 1/256, the 0.70703125 value makes sense; but that is context a new reader is not going to have necessarily, being forced to acquire that is a maintenance cost. The 0.70703125 looks to be a magic number code smell to me, can be fixed in a number of ways. In this case I would assign the 1/256 to a constant and maybe multiply it out, or add it as a named variable, or maybe a second look at the test method name. The existing name did not communicate to me that we are also normalizing to multiples of 1/256.

(side-note: Is this a SRP violation, we are both bounding and normalizing the input value, should these be done in two different steps with their own respect tests?)